lms/djangoapps/branding/api.py

```diff
@@ -569,22 +568,43 @@ def get_logo_url(is_secure=True):
         is_secure (bool): If true, use HTTPS as the protocol.
     """
 
-    # if the configuration has an overide value for the logo_image_url
+    # if the configuration has an override value for the logo_image_url
     # let's use that
-    image_url = configuration_helpers.get_value('logo_image_url')
-    if image_url:
-        return _absolute_url_staticfile(
-            is_secure=is_secure,
-            name=image_url,
-        )
+    logo_url = settings.LOGO_URL
+    default_local_path = 'images/logo.png'
+    logo_image_path = configuration_helpers.get_value('logo_image_url')
+
+    if logo_image_path:
+        return _absolute_url_staticfile(is_secure=is_secure, name=logo_image_path)
 
     # otherwise, use the legacy means to configure this
     university = configuration_helpers.get_value('university')
```

Comment:
Do you know what this `university` configuration setting is?  Is this something that is enabled on edx.org production?  If so, this won't work for Logo-Flipping. If not, I suppose it's fine to keep.

Reply:
`university` is not enabled on edx.org, just checked in the read replica,  I suppose it's fine to keep.